contrib/lib/src/lib.rs

```diff
@@ -81,3 +82,19 @@ mod uuid;
 
 #[cfg(feature = "uuid")]
 pub use uuid::{Uuid, UuidParseError};
+
+#[cfg(feature = "databases")]
+pub mod databases;
+
+#[cfg(feature = "databases")]
+#[allow(unused_imports)]
+#[macro_use]
+extern crate rocket_contrib_codegen;
+
+#[cfg(feature = "databases")]
+#[doc(hidden)]
+pub use rocket_contrib_codegen::*;
+
+pub fn log_err<'a>(msg: &'a str) {
```

Comment:
You don't need to write the lifetime explicitly here: `pub fn log_err(msg: &str)`. Also, should this function go in `rocket_contrib` or `rocket` core?

Reply:
Yep, I think it makes most sense in `rocket` core in the `log` crate.